### ISSUE TITLE
Property names are private

### ISSUE THREAD

**Issue by neilstevenson** [hazelcast/hazelcast-wm#67]
In https://github.com/hazelcast/hazelcast-wm/blob/master/src/main/java/com/hazelcast/web/WebFilterConfig.java#L47 the property names are private strings

Means we have to do

```
    properties.put("use-client", "true");
```

If we could do
```
    properties.put(WebFilterConfig.USE_CLIENT, "true");
```

then it's less error prone.

This would be consistent with the approach of the main Hazelcast codebase, such as https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java#L64

**Comment by vivekvara** [comment]
I found it useful for future releases and deprecation alerts. 

I would like to pick this up.  Can someone confirm?

**Approach:**  All constants should be public OR we can use patterns the same as KubernetesProperties.java

**Comment by enozcan** [comment]
Hi @vivekvara,
Thanks for that! I'm assigning you.
Please ask if you have any question.

**Comment by enozcan** [comment]
Closed by #91
